async-stream/Cargo.toml

```diff
@@ -9,18 +9,18 @@ name = "async-stream"
 #   - Cargo.toml
 #   - README.md
 # - Create git tag
-version = "0.3.0"
+version = "0.3.1"
 edition = "2018"
 license = "MIT"
 authors = ["Carl Lerche <me@carllerche.com>"]
 description = "Asynchronous streams using async & await notation"
-documentation = "https://docs.rs/async-stream/0.3.0/async_stream"
+documentation = "https://docs.rs/async-stream/0.3"
 homepage = "https://github.com/tokio-rs/async-stream"
 repository = "https://github.com/tokio-rs/async-stream"
 readme = "README.md"
 
 [dependencies]
-async-stream-impl = { version = "0.3.0", path = "../async-stream-impl" }
+async-stream-impl = { version = "=0.3.1", path = "../async-stream-impl" }
```

Comment:
async-stream-impl is the implementation detail of async-stream, using `=` requirements would help to avoid some problems like https://github.com/rust-lang/futures-rs/issues/2061